cmd/crc/cmd/root.go

```diff
@@ -37,14 +35,14 @@ func init() {
 	config.InitViper()
 	setConfigDefaults()
 	rootCmd.AddCommand(cmdConfig.ConfigCmd)
-	rootCmd.PersistentFlags().StringVar(&logLevel, "log-level", "info", "log level (e.g. \"debug | info | warn | error\")")
+	rootCmd.PersistentFlags().StringVar(&constants.LogLevel, "log-level", "info", "log level (e.g. \"debug | info | warn | error\")")
 }
 
 func runPrerun() {
 	output.OutF("%s - %s\n", commandName, descriptionShort)
 
 	// Setting up logrus
-	log.InitLogrus(logLevel)
+	log.InitLogrus(constants.LogLevel)
```

Comment:
... this is used as a shared value. This is not right. 

The purpose of the machine dir as-is can be interpreted as a const, as during use of the applciation this value will NOT change. But the `LogLevel` gets changed from a default to the actual value. If so, this is not the same behaviour.

Reply:
Shouldn't such a value make more sense to live in `logging.LogLevel` and use `constants.DefaultLogLevel` for an initial value in the `cmd` ?